c#-selenium-webdriver-sample/WebdriverFactory.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using OpenQA.Selenium;
+
+namespace CSharpSeleniumWebdriverSample
+{
+    public class WebdriverFactory
+    {
+        public static IWebDriver GetWebdriver(BrowserType browser)
+        {
+            IWebdriver webdriver = null;
+
+            switch (browser)
+            {
+                //In case using Chrome Web Driver
```

Comment:
I think the comments above these case blocks can be omitted; they aren't adding any new information except what the case statement already says